data-raw/R2R_TMH_habitat_inputs/documentation/TMH_methodology.Rmd

```diff
@@ -0,0 +1,178 @@
+---
+output:
+  html_document:
+    theme: paper
+    df_print: paged
+  pdf_document: default
+  word_document: default
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+## Theoretical Maximum Habitat Methodology
+
+### Purpose
+
+Set a maximum abundance target through estimating maximum habitat potential - habitat isn't directly proportional to abundance but is a key parameter in defining population carrying capacity in the DSM estimate of maximum suitable area of spawning, in-channel rearing, and floodplain rearing habitat for salmonids in all watersheds included in the CVPIA Decision Support Models (DSMs). The main objectives for theoretical maximum habitat calculations are to:
+
+-   Include potential suitable habitat area downstream and upstream of dams
+
+-   Include potential suitable habitat area within and beyond levees
+
+### Methodology
+
+The theoretical maximum suitable habitat for spawning, instream rearing and floodplain rearing was estimated for Fall Run Chinook. For reaches with major rim dams, habitat values were estimated for above and below the dam. Theoretical maximum habitat values were calculated for most watersheds using a geospatial analysis and for a few watersheds using a HEC RAS model.
```

Comment:
Please describe which watersheds you use each methodology - maybe a table at the end of the doc

Reply:
added an appendix and table